docs/dev-guide/contributing/bugs.rst

```diff
@@ -65,6 +65,10 @@ Developer
    is included in the merge request, as well as a brief description of what the change is. It is
    not required to find and assign someone to do the review.
 #. When the pull request is submitted, the developer changes the status of the bug to ``POST``.
+
+.. note::
```

Comment:
I think this should not be a note style. This is a required step just like the list of steps above. Consider rewriting the step above from:

```
the developer changes the status of the bug to ``POST``.
```

=>

```
the developer changes the status of the bug to ``POST`` and sets the appropriate target release.
```